### ISSUE TITLE
The landmark loss reduces slowly.

### ISSUE THREAD

**Issue by AndrewDing616** [ElvishElvis/68-Retinaface-Pytorch-version#21]
Hello, ElvishElvis.
Thank you for your great work!
I change the backbone to mobilenet_v3 and regnet, and use the LS3D-W dataset as my training dataset. But the landmark branch loss reduces slowly (using smoothL1 Loss), for example, 1.79 -> 1.77 after 20 epochs, while the other losses reduces normally. 
So, have you met this problem? And could you share your training strategy if possible?
